src/libraries/Common/src/System/Net/Http/WinInetProxyHelper.cs

```diff
@@ -117,13 +117,25 @@ public WinInetProxyHelper()
             //    fAutoLogonIfChallenged member set to TRUE.
             //
             // We match behavior of WINHTTP_ACCESS_TYPE_AUTOMATIC_PROXY and ignore errors.
+
+            string destination = uri.AbsoluteUri;
+            // Underlying code does not understand WebSockets so we need to convert it to http or https.
+            if (uri.Scheme == UriScheme.Wss)
+            {
+                destination = "https" + destination.Substring(3);
+            }
+            else if (uri.Scheme == UriScheme.Ws)
+            {
+                destination = "http" + destination.Substring(2);
```

Comment:
Does this need to compile into any non-netcoreapp projects?  If it's just netcoreapp, it could be:
```C#
destination = string.Concat("http", destination.AsSpan(2));
```
to save a string allocation (and same with similar case above).

Reply:
I will check. I think this is shared with WinHttp package but I'm not sure what they target.

Reply:
it seems like WinHttp targets netstandard2.0 and possibly net461. (the configuration is there but I'm not sure if we really build it) 
Is this worth of something like `#if NETCOREAPP` ? (or the other way)

Reply:
Thanks. Probably not worth it if we're talking one string per ClientWebSocket.